torch/__init__.py

```diff
@@ -1650,3 +1650,7 @@ def _sparse_coo_tensor_unsafe(*args, **kwargs):
 
 from . import _logging
 _logging._init_logs()
+
+# Register functional collectives
+import torch.distributed._functional_ops as _functional_ops
+del _functional_ops
```

Comment:
im not sure what the point of this is, probably need a comment here

Reply:
The point is ensuring no namespace pollution.
We want make sure that nobody can do `import torch._functional_ops as big_mistake`

Reply:
i see, yea that makes sense. you're just doing this to trigger the init for deploy, iiuc